viz/charts/linkerd-viz/Chart.yaml

```diff
@@ -11,7 +11,7 @@ kubeVersion: ">=1.21.0-0"
 name: "linkerd-viz"
 sources:
 - https://github.com/linkerd/linkerd2/
-version: 30.3.3
+version: 30.3.4
```

Comment:
Our release process is that we only increment this version during the release.  So this should stay at 30.3.3 in this PR.